vexim/config/httpheaders.php

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
Maybe you have output buffering enabled. I don't remember if that is the default, but I'm quite sure it can be turned off.
The alternative is probably to use `ob_start` and `ob_flush`, but you'd still have to edit almost all of our php scripts to either include a "footer" script of some sort, or to call `ob_flush` explicitly. At this point, just adding a doctype where necessary might be easier (or leaving it out since everything works anyway and we'll be killing all of this markup in our next release anyway).